cpu/stm32f1/include/cpu_conf.h

```diff
@@ -36,12 +37,122 @@ extern "C" {
 #endif
 
 /**
+ * @brief   Flash page configuration
```

Comment:
how about factoring the firmware upgrade stuff out of cpu_conf.h?

CPU specific defines, like ```FLASHPAGE_*```, could stay.